pkg/proto/nbcontract/v1/config.proto

```diff
@@ -15,21 +15,24 @@ import "pkg/proto/nbcontract/v1/kubeletconfig.proto";
 import "pkg/proto/nbcontract/v1/featurestate.proto";
 import "pkg/proto/nbcontract/v1/teleportconfig.proto";
 import "pkg/proto/nbcontract/v1/runcconfig.proto";
-import "pkg/proto/nbcontract/v1/swapfileconfig.proto";
+import "pkg/proto/nbcontract/v1/customcloudconfig.proto";
+import "pkg/proto/nbcontract/v1/customcatrustconfig.proto";
+import "pkg/proto/nbcontract/v1/apiserverconfig.proto";
+import "pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto";
+import "pkg/proto/nbcontract/v1/nodepoolprofile.proto";
 
 // illustrative division only
 message Configuration {
   // System Configuration
   
   string cluster_certificate_authority = 1;
-
-  // only required until Secure TLS bootstrapping in place
-  // would use kubelet identity after that.
-  string tls_bootstrap_token = 2;
+  TLSBootstrappingConfig tls_bootstrapping_config = 2;
+  
 
   // cluster/user config
   string kubernetes_version = 3; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
   KubeBinaryConfig kube_binary_config = 4;
+  optional CustomCloudConfig custom_cloud_config = 5;
```

Comment:
At some point lets review for consistent use (or not) of *Config and _config suffixes

Reply:
Sounds good. For now I try to add _config suffix for sub-groups. But can definitely review this once the contract is more stable.

Reply:
Oh I know why we needed to keep the prefix.
For example, if I put

`enum NetworkPluginType {
  UNSPECIFIED = 0;
  NONE = 1;
  AZURE = 2;
  KUBENET = 3;
}

enum NetworkPolicyType {
  UNSPECIFIED = 0;
  NONE = 1;
  AZURE = 2;
  CALICO = 3;
}`


It will complain this.
`symbol "nbcontract.v1.AZURE" already defined at pkg/proto/nbcontract/v1/networkconfig.proto:32:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum`